constants/members.json

```diff
@@ -3429,6 +3437,111 @@
+    },
+    {
+      "name": "William Park",
+      "role": "developer",
+      "term": 1236,
+      "teams": ["fck"],
+      "img": "https://firebasestorage.googleapis.com/v0/b/uw-blueprint.appspot.com/o/img%2Ff23headshosts%2FDev_William_Park.jpeg?alt=media&token=019bc1c3-95d4-4ee7-8c4d-bfb3bd1697c1"
+    },
+    {
+      "name": "Sunny Zhang",
+      "role": "designer",
+      "term": 1236,
+      "teams": ["fck"],
+      "img": "https://firebasestorage.googleapis.com/v0/b/uw-blueprint.appspot.com/o/img%2Ff23headshosts%2FProductDesigner_Sunny_Zhang.JPG?alt=media&token=b935d1a8-174d-442a-86e4-f8d0563c8b78"
+    },
+    {
+      "name": "Daniel Truong",
+      "role": "developer",
+      "term": 1236,
+      "teams": ["mp"],
+      "img": "https://firebasestorage.googleapis.com/v0/b/uw-blueprint.appspot.com/o/img%2Ff23headshosts%2FDev_Daniel_Truong?alt=media&token=963bf59c-6ab3-4d35-98b7-c27f89189177"
+    },
+    {
+      "name": "Jesse Huang",
+      "role": "developer",
+      "term": 1236,
+      "teams": ["mp"],
+      "img": "https://firebasestorage.googleapis.com/v0/b/uw-blueprint.appspot.com/o/img%2Ff23headshosts%2FDev_Jesse_Huang.JPG?alt=media&token=b484c2a5-d2c4-4ebc-b1f9-46cfdfbac5a6"
+    },
+    {
+      "name": "Janie Ong",
+      "role": "designer",
+      "term": 1236,
+      "teams": ["mp"],
+      "img": "https://firebasestorage.googleapis.com/v0/b/uw-blueprint.appspot.com/o/img%2Ff23headshosts%2FProductDesigner_Janie_Ong.JPG?alt=media&token=06f55dcd-299b-490d-9ee9-d4d6a82afd45"
+    },
+    {
+      "name": "Shelley Chen",
+      "role": "developer",
+      "term": 1236,
+      "teams": ["mp"],
+      "img": "https://firebasestorage.googleapis.com/v0/b/uw-blueprint.appspot.com/o/img%2Ff23headshosts%2FDev_Shelley_Chen.jpeg?alt=media&token=ac5f232d-1c1e-4ff9-85e2-85bbebfd8c3c"
+    },
+    {
+      "name": "Victoria Ngyugen",
+      "role": "vp talent",
+      "term": 1236,
+      "teams": ["executive"],
+      "img": "https://firebasestorage.googleapis.com/v0/b/uw-blueprint.appspot.com/o/img%2Fdefault.png?alt=media&token=fe95cc90-ba2b-4c04-a808-0f903cc8b519"
+    },
+    {
+      "name": "Janaki Menon",
+      "role": "vp talent",
+      "term": 1236,
+      "teams": ["executive"],
+      "img": "https://firebasestorage.googleapis.com/v0/b/uw-blueprint.appspot.com/o/img%2Fdefault.png?alt=media&token=fe95cc90-ba2b-4c04-a808-0f903cc8b519"
+    },
+    {
+      "name": "Parth Desai",
+      "role": "vp scoping",
+      "term": 1236,
+      "teams": ["executive"],
+      "img": "https://firebasestorage.googleapis.com/v0/b/uw-blueprint.appspot.com/o/img%2Fdefault.png?alt=media&token=fe95cc90-ba2b-4c04-a808-0f903cc8b519"
+    },
+    {
+      "name": "Mathily Kadam",
+      "role": "internal director",
+      "term": 1236,
+      "teams": ["internal director"],
```

Comment:
Where is the `internal director` team? I think the role `internal director` should be under `teams: executive`. This applies for `Olivia Chan`, `Maggie Lu`, `Krithi Sodhi`, `Serena Poonawalla` as well. Thoughts?